client/SDL/sdl_freerdp.cpp

```diff
@@ -612,29 +609,53 @@ static BOOL sdl_create_windows(SdlContext* sdl)
 	auto title = sdl_window_get_title(sdl->context()->settings);
 	BOOL rc = FALSE;
 
-	// TODO: Multimonitor setup
-	const size_t windowCount = 1;
+	UINT32 windowCount =
+	    freerdp_settings_get_uint32(sdl->context()->settings, FreeRDP_MonitorCount);
 
-	const UINT32 w = freerdp_settings_get_uint32(sdl->context()->settings, FreeRDP_DesktopWidth);
-	const UINT32 h = freerdp_settings_get_uint32(sdl->context()->settings, FreeRDP_DesktopHeight);
-
-	for (size_t x = 0; x < windowCount; x++)
+	for (UINT32 x = 0; x < windowCount; x++)
 	{
+		auto monitor = static_cast<rdpMonitor*>(freerdp_settings_get_pointer_array_writable(
+		    sdl->context()->settings, FreeRDP_MonitorDefArray, x));
+
+		Uint32 w = monitor->width;
+		Uint32 h = monitor->height;
+		if (!(sdl->context()->settings->UseMultimon || sdl->context()->settings->Fullscreen))
```

Comment:
we should alias  `sdl->context()->settings` that is used multiple times here

Reply:
done